src/mon/OSDMonitor.cc

```diff
@@ -97,7 +97,7 @@ void OSDMonitor::_get_pending_crush(CrushWrapper& newcrush)
   if (pending_inc.crush.length())
     bl = pending_inc.crush;
   else
-    osdmap.crush->encode(bl);
+    osdmap.crush->encode(bl, CEPH_FEATURES_SUPPORTED_DEFAULT);
```

Comment:
It turns out the features for the pending OSDMap::Incremental don't matter.  The canonical mon map and the osdmap full map both encode from OSDMap.. they don't use the encoded crush in the incremental.